client/src/components/Pages/Home/constants.js

```diff
@@ -79,6 +81,7 @@ export const hobbies = [
       for other people. For the moment, the most pleasure I have working
       with Javascript, more precisely with Node and React.`,
     id: 'hobby-1',
+    storageName: 'programming',
```

Comment:
I know, probably this is not the best solution, but for the moment I want to keep it like this. Hopefully, soon I will have time for refactoring and I will find a better way.